src/daemon/metric.c

```diff
@@ -41,13 +43,64 @@
 /* Metric names must match the regex `[a-zA-Z_:][a-zA-Z0-9_:]*` */
 #define VALID_NAME_CHARS VALID_LABEL_CHARS ":"
 
+int distribution_count_marshal_text(strbuf_t *buf, distribution_t *dist) {
+
```

Comment:
In your opinion: deleting these empty lines before after function name would make code more readable.

Reply:
Thank you Barbara for your suggestion, I agree with you.